pkg/api/annotations_test.go

```diff
@@ -950,3 +957,139 @@ func setUpFGACGuardian(t *testing.T) {
 
 	guardian.MockDashboardGuardian(&guardian.FakeDashboardGuardian{CanEditValue: true})
 }
+
+// Note: this is a database test, so it should be located in pkg/services/sqlstore/annotation_test.go
+// however, placing it there would currently result in circular dependencies.
+// So this test should be moved once the SQL package is separated out.
```

Comment:
nit : I guess this is okay, but should not we resolve circular dependencies instead. It's so incredible that we have a test for it

do we have any plans to move this into an `annotation` package?

Reply:
I think abstractions team are working on restructuring the code, which should fix the circular dependencies.

Agreed that it would be a better fix, but didn't want to introduce too many changes in this PR. I was thinking about raising a follow up PR to move annotations to their own package next week if I have time.

Reply:
You can use `package sqlstore_test` to place this in another package within sqlstore for the sake of not running into circular dependencies, see https://pkg.go.dev/cmd/go@master#hdr-Test_packages

Reply:
Oh thanks, that's a great suggestion! I've moved the tests and renamed the package for annotation SQL tests.